feature-libs/cart/base/components/added-to-cart-dialog/added-to-cart-dialog.component.html

```diff
@@ -1,124 +1,129 @@
-<div #dialog>
```

Comment:
Let's include information about removing one layer of wrappers from this component in the migration docs. You can follow the pattern and province `#### Template changes` as a paragraph to mention those changes.